tests/cephadm/test_remove_cluster.py

```diff
@@ -105,6 +107,11 @@ def run(ceph_cluster, **kw):
                 f"Failed to execute rm-cluster on node '{node.ip_address}'"
             )
 
+    # Check if validation set to `False`
+    if not validation:
```

Comment:
can we cross check if the validation variable is str or bool

Reply:
Done